From 9ec3449788742b5f93a02c154a18427602ace8e6 Mon Sep 17 00:00:00 2001
From: renchenglei <chenglei.ren@intel.com>
Date: Thu, 9 Sep 2021 16:18:41 +0800
Subject: [PATCH] INTERNAL: Add more usage for video format

This is to help add more usage for NV12
and P010 format

Tracked-On: OAM-99322
Signed-off-by: Chenglei Ren <chenglei.ren@intel.com>
---
 i915_private.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/i915_private.c b/i915_private.c
index b800d2ba27f0..9696b8dd46d6 100644
--- a/i915_private.c
+++ b/i915_private.c
@@ -102,7 +102,7 @@ int i915_private_add_combinations(struct driver *drv)
 	metadata.priority = 3;
 	metadata.modifier = I915_FORMAT_MOD_Y_TILED;
 	drv_add_combinations(drv, private_source_formats, ARRAY_SIZE(private_source_formats),
-			     &metadata, texture_flags | BO_USE_CAMERA_MASK);
+			     &metadata, texture_flags | BO_USE_NON_GPU_HW);
 
 	texture_flags &= ~BO_USE_RENDERSCRIPT;
 	texture_flags &= ~BO_USE_SW_WRITE_OFTEN;
-- 
2.17.1

